Bottom: 1fc6063e22d36d7626822b9d0af991982aa57675
Top:    9ad1cfb330fa05dbd1459c4ac84c4aa4a10be3cc
Author: scott snyder <sss@karma>
Date:   2020-02-12 04:07:29 -0500

Refresh of gitignore

---

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..b25c15b
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+*~
